Common/Utils/include/CommonUtils/ShmManager.h

```diff
@@ -28,7 +28,7 @@
 
 #if !defined(__CINT__) && !defined(__MAKECINT__) && !defined(__ROOTCLING__) && !defined(__CLING__)
 // this shared mem mode is meant for compiled stuff in o2-sim; not for ROOT sessions
-#define USESHM 1
+//#define USESHM 1
```

Comment:
We can probably turn it off whenever the platform is Apple.

Reply:
That's fine with me. Notice however that clang seems to be right in complaining about the way the rebinding of the allocator happens, so in the long term it will probably need a fix also on linux.